packages/opencensus-web-all/src/initial-load-context.ts

```diff
@@ -21,27 +21,37 @@ import {WindowWithOcwGlobals} from './types';
 
 const windowWithOcwGlobals = window as WindowWithOcwGlobals;
 
+/**
+ * The default trace sampling rate if no `traceparent` and no `ocSampleRate`
+ * are specified on the `window`.
+ */
+const DEFAULT_SAMPLE_RATE = 0.001;
```

Comment:
FYI in other languages the default is 1/10000.

Reply:
Thanks for pointing that out. Updated to `0.0001` (1/10000) to match the other languages.